Makefile

```diff
@@ -218,16 +250,16 @@ proto/felixbackend.pb.go: proto/felixbackend.proto
 ###############################################################################
 CONTAINER_CREATED=.dikastes.created-$(ARCH)
 .PHONY: image calico/dikastes
```

Comment:
It would be good to update this one last calico/dikastes reference to $(BUILD_IMAGE)

Reply:
Yup. Fixing now.